_about/notes/0.5.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Control Plane Deployments**. `istioctl` now has a deployment command to produce istio control plane deployments. TODO: Doc link?
+
+- **Automatic Proxy Injection**. Use Kubernetes 1.9's mutating webhook functionality to provide automatic
+pod-level proxy injection. Automatic injection requires Kubernetes 1.9 or beyond and
+therefore doesn't work on older versions. [Learn more]({{home}}/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection)
+
+- **Alpha2 Traffic Rules**. We've started to add support for a new model for routing and destination rules. You can see the updated
+definitions [here]({{home}}/docs/reference/config/istio.routing.v1alpha2.html). It's not quite ready to be used though, this is
+still very much a work in progress.
+
+## Mixer adapters
+
+- **Open Policy Agent**. Mixer now has an authorization adapter implementing the [open policy agent](http://www.openpolicyagent.org) model,
+providing a flexible fine-grained access control mechanism. TODO: Doc link?
+
+- **RBAC**. Mixer now has a resource-based access control adapter. TBD: Doc link?
+
+- **Fluentd**. Mixer now has an adapter for log collection through [fluentd](https://www.fluentd.org). 
+[Learn more]({{home}}/docs/tasks/telemetry/fluentd.html)
+
+- **Stdio**. The stdio adapter now lets you log to files with support for log rotation & backup, along with a host
+of controls.
+
+## Security
+
+- **Bring Your Own CA**. There have been many enhancements to the 'bring your own CA' feature. TODO: Doc link?
```

Comment:
Do we have a doc link for this or any more detail we can list directly in this doc?

Reply:
https://istio.io/docs/tasks/security/plugin-ca-cert.html